ajax/libs/jquery.adaptive-backgrounds/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "author": "Brian Gonzalez",
+  "name": "jquery.adaptive-backgrounds",
+  "description": "A simple jQuery plugin to extract the dominant color of an image and apply it to the background of its parent element.",
+  "version": "1.0.3",
+  "repository": {
+    "type": "git",
+    "url": "http://briangonzalez.github.io/jquery.adaptive-backgrounds.js/"
```

Comment:
We use GitHub repo url here usually.
Also this link is as the same with homepage and I think it's fine let this link as the homepage. (Although it will different from upstream, I think it's OK.)
So let's modify this field as GitHub link, what do you think?
Thanks.